rudder-web/src/main/scala/com/normation/rudder/web/rest/RestUtils.scala

```diff
@@ -94,7 +94,7 @@ object RestUtils extends Loggable {
    *   - Add a new line at the end and beginning of an array and indent one more level array data
    *   - space after colon
    */
-  def render(value: JValue): Document = {
+  def rendrrer(value: JValue): Document = {
```

Comment:
hum - there is something fishy there: you change a method name, but the comment is not updated
link should probably be 
https://github.com/lift/framework/blob/2.6.3-release/core/json/src/main/scala/net/liftweb/json/JsonAST.scala#L406